apps/web-giddh/src/app/shared/subscription-upgrade-button/subscription-upgrade-button.component.ts

```
@@ -61,11 +63,11 @@ export class SubscriptionUpgradeButtonComponent implements OnDestroy {
      * @memberof SubscriptionUpgradeButtonComponent
      */
     public shouldShowMessage(): boolean {
-        const hasRestriction = this.activeCompany?.subscription?.planDetails?.restrictedModules.hasOwnProperty(this.restrictedModule);
-        if (this.remainingCount !== null) {
-            return this.remainingCount === 0 && hasRestriction;
+        if (this.restrictedModule === this.restrictedModules.Users) {
+            return this.isUserRestricted;
+        } else {
+            return Object.hasOwn(this.activeCompany?.subscription?.planDetails?.restrictedModules, this.restrictedModule);;
```

Biome (1.9.4)

[error] This code is unreachable

... because this statement will return from the function beforehand


(lint/correctness/noUnreachable)